Cakefile

```diff
@@ -146,7 +146,7 @@ task 'build:browser', 'rebuild the merged script for inclusion in the browser',
   b.add "./lib/coffee-script/browser.js"
   await b.bundle { standalone : 'CoffeeScript' }, defer err, code
   console.error err if err?
-  code = "(function(root){\n" + code + "\nroot.CoffeeScript = CoffeeScript;\n})(this);"
```

Comment:
Do we want to avoid executing the whole closure if root.Coffeescript is already defined, i.e.?

if (!this.CoffeeScript) (function(root){....})(this);